cg/meta/compress/compress.py

```diff
@@ -112,6 +112,11 @@ def decompress_spring(self, sample_id: str) -> bool:
         for compression in compressions:
             if not self.crunchy_api.is_spring_decompression_possible(compression_obj=compression):
                 LOG.info(f"SPRING to FASTQ decompression not possible for {sample_id}")
+                if self.crunchy_api.is_compression_pending(compression):
+                    LOG.info(
+                        f"Spring file {compression.spring_path.as_posix()} is already being decompressed."
+                    )
+                    continue
                 if not self.backup_api.is_to_be_retrieved_and_decrypted(
                     spring_file_path=compression.spring_path
                 ):
```

Comment:
Im also not sure of this `return False`, wouldn't it be better a `continue`? An the following log is also confusing for me. Maybe we can fix that in this PR too, what do you think? @islean

Reply:
I think the logging message could be improved in this PR but would a continue not eventually make the return statement True, even if some files are not on Hasta? @diitaz93 ?

Reply:
Although I guess it is intended as a double "SPRING to FASTQ decompression not possible for {sample_id}" + "Until the SPRING file is retrieved from PDC and decrypted"

Reply:
Changed to `LOG.info("The SPRING file will be retrieved from PDC and decrypted")`

Reply:
👍